docs/repositories-coverage/commits.md

```diff
@@ -76,3 +76,23 @@ The option **Show also files without coverage changes** allows you to list all f
 ![Files tab](images/{{ page.meta.file_name }}-tab-files.png)
 <!-- vale on -->
 <!--tab-files-end-->
+
+<!--tab-diff-start-->
+## Diff tab
+
+The **Diff** tab displays a line-by-line view of the coverage variation introduced by the {{ page.meta.page_name }}. It includes the following areas:
+
+-   A **list of files** modified by the {{ page.meta.page_name }}, with additional information for each file:
+
+    -   A **green plus icon** if the file is added or a **yellow dot icon** if it's modified by the {{ page.meta.page_name }}
+    -   The **coverage variation** introduced by the {{ page.meta.page_name }} or the **total file coverage** if there's no variation
+
+-   A **diff viewer** comparing the old and new content of each modified file. The background of any added or modified coverable lines depends on their coverage status:
```

Comment:
Shall we also mention here the diff coverage metric present in the file's diff header?